packages/SwingSet/test/vat-admin/terminate/terminate.test.js

```diff
@@ -471,6 +471,23 @@ test.serial('dead vat state removed', async t => {
   t.is(kvStore.get('vat.dynamicIDs'), '["v6"]');
   t.is(kvStore.get('ko26.owner'), 'v6');
   t.is(Array.from(enumeratePrefixedKeys(kvStore, 'v6.')).length > 10, true);
+  // find the two promises in the new vat's c-list, they should both
+  // have refCount=2 (one for bootstrap, one for the dynamic vat)
+  let decidedKPID;
+  let subscribedKPID;
+  for (const key of enumeratePrefixedKeys(kvStore, 'v6.')) {
+    if (key.startsWith('v6.c.kp')) {
+      const kpid = key.slice('v6.c.'.length);
+      const refCount = Number(kvStore.get(`${kpid}.refCount`));
+      const decider = kvStore.get(`${kpid}.decider`);
+      if (decider === 'v6') {
+        decidedKPID = kpid;
+      } else {
+        subscribedKPID = kpid;
+      }
+      t.is(refCount, 2, `${kpid}.refCount=${refCount}, not 2`);
+    }
+  }
```

Comment:
I noticed in the other file that we pretty much always need to strip off the prefix for each key returned from `enumeratePrefixedKeys` (which makes sense). If we refactored it to produce `{ key, suffix }` records (where suffix has had the prefix stripped), then we'd end up here with something like
```suggestion
  for (const { suffix } of enumeratePrefixedKeys(kvStore, 'v6.c.kp')) {
    const kpid = `kp${suffix}`;
    const refCount = Number(kvStore.get(`${kpid}.refCount`));
    const decider = kvStore.get(`${kpid}.decider`);
    if (decider === 'v6') {
      decidedKPID = kpid;
    } else {
      subscribedKPID = kpid;
    }
    t.is(refCount, 2, `${kpid}.refCount=${refCount}, not 2`);
  }
```

Reply:
filed #10340 to remember to do this refactoring later